myapp/Basic AutoLayout/SignUpViewController.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  SignUpViewController.swift
+//  myapp
+//
+//  Created by Stuti Bhavsar  on 18/04/22.
+//
+
+import UIKit
+
+class SignUpViewController: UIViewController {
+    var activeField: UITextField?
+    var lastOffset: CGPoint!
```

Comment:
why use ! for variable?
Code formation should be first outlets, then variables and proper mark should be there